tests/conftest.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from bird_cloud_gnn.fake import generate_data
+from bird_cloud_gnn.radar_dataset import RadarDataset
+import pytest
+
+
+@pytest.fixture()
+def setup_dataset():
+    """Setup radar dataset"""
+    with TemporaryDirectory() as tmp:
+        tmp_path = Path(tmp)
```

Comment:
Is it possible to use the existing `tmp_path`? Or does this `TemporaryDirectory` returns the same path? Because, otherwise, we won't know where the files are. On the other hand, we might not need it.

Reply:
I created an issue to investigate this.

Reply:
Thank you